source/common/http/filter/lua/lua_filter.cc

```diff
@@ -117,7 +117,7 @@ int StreamHandleWrapper::luaRespond(lua_State* state) {
   uint64_t status;
   if (headers->Status() == nullptr ||
       !StringUtil::atoul(headers->Status()->value().c_str(), status) || status < 200 ||
-      status >= 500) {
+      status >= 600) {
```

Comment:
Thanks. What about the `status < 200` clause and claim in error that the range `100-599`?